tests/functional/src/main/java/org/mule/functional/classloading/isolation/classloader/IsolatedClassLoaderFactory.java

```diff
@@ -125,20 +132,43 @@ protected ArtifactClassLoader createContainerArtifactClassLoader(TestContainerCl
   }
 
   /**
-   * For each plugin defined in the classification it will create an {@link ArtifactClassLoader} with the name defined in
-   * classification. For extension plugins it will also create the filter based on the extension manifest file. For a plain plugin
-   * it will collect the exported packages and resources for creating the filter from its mule-module.properties file. <pr/> It
-   * also creates a sharedLibs plugin without any library so far, in order to be a similar representation as mule's container has
-   * and also to allow the hierarchy process to look for classes from its parent (see {@link CompositeClassLoader} that doesn't
-   * delegate to its parent). <pr/> With the given list of created class loader plugins it will finally create a
-   * {@link CompositeClassLoader} and return it.
+   * For each service defined in the classification it creates an {@link ArtifactClassLoader} wit the name defined in
+   * classification.
+   *
+   * @param parent the parent class loader to be assigned to the new one created here
+   * @param childClassLoaderLookupPolicy look policy to be used
+   * @param artifactUrlClassification the url classifications to get plugins {@link URL}s
+   * @return a {@link CompositeClassLoader} that represents the service class loaders
+   */
+  protected List<ArtifactClassLoader> createServiceClassLoaders(ClassLoader parent,
+                                                                ClassLoaderLookupPolicy childClassLoaderLookupPolicy,
+                                                                ArtifactUrlClassification artifactUrlClassification) {
```

Comment:
Not sure that ArtifactUrlClassification is the right name, looks more like a ArtifactsUrlClassification